KoreBuild-dotnet/build/_xunit-test.shade

```diff
@@ -3,6 +3,7 @@ use import="Environment"
 
 default NO_PARALLEL_TEST_PROJECTS='${E("NO_PARALLEL_TEST_PROJECTS")}'
 default KOREBUILD_TEST_SKIPMONO='${E("KOREBUILD_TEST_SKIPMONO")}'
+default TEST_ARTIFACT_DIR='${Path.Combine(Directory.GetCurrentDirectory(), "artifacts", "tests")}'
```

Comment:
Good idea. I'll change to publish to `obj/` so we don't need to update filepaths in testing and then just copy the files to the artifacts directory. This is what we do for all other artifacts anyways.